arch/platform/cc2538dk/platform.c

```diff
@@ -154,6 +155,9 @@ platform_init_stage_two()
   crypto_init();
   crypto_disable();
 #endif
+#if CSPRNG_ENABLED
+  csprng_init();
```

Comment:
Agree with @simonduq This should go to `main()`.  `cc2538-conf.h` should then `#define CSPRNG_CONF_ENABLED 1`.

Reply:
This went to contikimac-main.c. I have not enabled CSPRNG_CONF_ENABLED by default because it may not be needed, but I have no objection to enabling it by default.